archived/star-rx72n-firmware/tests/test_rx_drv8243_spi.c

```
@@ -0,0 +1,1172 @@
+  if (err != k_rx_ok) {
+    return err;
+  }
+
+  h->last_spi_status = DRV8243_SPI_GET_STATUS(rx_frame);
+  *data              = DRV8243_SPI_GET_DATA(rx_frame);
+
+  return k_rx_ok;
+}
+
+/** @brief Internal helper to write a register */
+static rx_err_t internal_drv8243_spi_write_reg(rx_drv8243_handle_t* h, uint8_t addr, uint8_t data)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'internal_drv8243_spi_write_reg' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)